include/zephyr/drivers/pcie/pcie.h

```diff
@@ -37,6 +37,16 @@ typedef uint32_t pcie_bdf_t;
  */
 typedef uint32_t pcie_id_t;
 
+/* Helper macro to exclude invalid PCIe identifiers. We should really only
```

Comment:
Should we document it like other macros?
```
/**
 * @brief ...
```

Reply:
I think this is really more of an internal API, only needed by `shell.c` and `pcie.c`. We might want to move such things to a new `drivers/pcie/host/pcie_internal.h` file or similar. Anyway, I guess it's a bit out of scope for this PR.